.evergreen/config.yml

```diff
@@ -363,7 +366,7 @@ functions:
         working_dir: "src"
         script: |
           ${PREPARE_SHELL}
-          TESTS=${TESTS} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" sh ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
+          TESTS=${TESTS} SSL=${SSL} MONGODB_URI="${MONGODB_URI}${APPEND_URI}" sh ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
```

Comment:
Will `${MONGODB_URI}` always omit the trailing slash and URI options, such that `${APPEND_URI}` is safe to concatenate?

I'm not sure where this was being handled previously. It looks like it was previously a `var` for `bootstrap mongo-orchestration`, but I didn't find how it was referenced in either this file or drivers-evergreen-tools.

Reply:
Not entirely. This was previously handled in run-orchestration.sh, but the default run-orchestration script in drivers-evergreen-tools does not support it. The main reason for this variable is to append SSL data, which is not included in the mongodb_uri provided by mongo-orchestration (see https://github.com/10gen/mongo-orchestration/issues/268). I plan to revisit this in mo directly in the coming weeks so we can get rid of this